parts/inc/Sv_set

```diff
@@ -101,10 +103,24 @@ __UNDEFINED__ sv_setsv_flags(dstr, sstr, flags)
   )
 #endif
 
-#if defined(PERL_USE_GCC_BRACE_GROUPS)
-__UNDEFINED__ newSVsv_flags(sv, flags) ({ SV *_sv = newSV(0); sv_setsv_flags(_sv, (sv), (flags)); _sv; })
-#else
-__UNDEFINED__ newSVsv_flags(sv, flags) ((PL_Sv = newSV(0)), sv_setsv_flags(PL_Sv, (sv), (flags)), PL_Sv)
+#ifndef newSVsv_flags
+#  if defined(PERL_USE_GCC_BRACE_GROUPS)
+#    define  newSVsv_flags(sv, flags)                       \
+        ({                                                  \
+            SV *new = newSV(0);                             \
+            sv_setsv_flags(new, (sv), (flags));             \
+            new;                                            \
+        })
+#  else
+    PERL_STATIC_INLINE SV* D_PPP_newSVsv_flags(SV *const old, I32 flags)
+        {
+            dTHX;
+            SV *new = newSV(0);
+            sv_setsv_flags(new, old, flags);
+            return new;
+        }
+#    define  newSVsv_flags(sv, flags) D_PPP_newSVsv_flags(sv, flags)
```

Comment:
same here why removing the usage of `__UNDEFINED__` ?

Reply:
I think it didn't work right due to the conditional compilation regarding brace groups.  But maybe there was something else wrong

Reply:
ok so are we happy with it like this? or do we want to try to put it back?